ecl/hthor/hthor.cpp

```diff
@@ -2494,14 +2494,62 @@ const void *CHThorGroupDedupKeepRightActivity::nextRow()
 
 //=====================================================================================================
 
+CHThorGroupDedupKeepBestActivity::CHThorGroupDedupKeepBestActivity(IAgentContext &_agent, unsigned _activityId, unsigned _subgraphId, IHThorDedupArg &_arg, ThorActivityKind _kind, bool _keepLeft)
+    : CHThorGroupDedupActivity(_agent, _activityId, _subgraphId, _arg, _kind), keepLeft (_keepLeft)
+{
+    compareBest = helper.queryCompareBest();
+}
+
+void CHThorGroupDedupKeepBestActivity::ready()
+{
+    CHThorGroupDedupActivity::ready();
+    assertex(numToKeep==1);
```

Comment:
I agree that it does make sense.  What should we do with this functionality: add a TODO comment, create a jira ticket to discuss later or go ahead and implement in this ticket?